src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -167,18 +175,33 @@ export default function FashionDrawer({
 
     const set = defs.PresentationNode.get(
       parseInt(mostCommonOrnamentSet[0], 10)
-    ).children.collectibles.map((c) => defs.Collectible.get(c.collectibleHash).itemHash);
+    ).children.collectibles.map(
+      (c) =>
+        defs.Collectible.get(c.collectibleHash).itemHash ??
+        manuallyFindItemForCollectible(defs, c.collectibleHash)
+    );
 
     setModsByBucket((modsByBucket) =>
       Object.fromEntries(
         LockableBucketHashes.map((bucketHash, i) => {
-          const ornament = set[i];
-          if (unlockedPlugs.has(ornament)) {
+          let ornamentHash = set[i];
+
+          // if we picked a parent node that doesn't point to ornaments,
+          // try to find an "identical" item that *is* an unlocked plug
+          if (!unlockedPlugs.has(ornamentHash)) {
+            const ornamentVersion = findOtherCopies(defs, ornamentHash).find((i) =>
+              unlockedPlugs.has(i.hash)
+            );
+            ornamentHash = ornamentVersion?.hash ?? ornamentHash;
+          }
+
+          if (unlockedPlugs.has(ornamentHash)) {
             const modsWithoutShaders = (modsByBucket[bucketHash] ?? []).filter((h) => isShader(h));
-            const mods = [...modsWithoutShaders, ornament];
+            const mods = [...modsWithoutShaders, ornamentHash];
             return [bucketHash, mods];
           }
-          return [bucketHash, modsByBucket[bucketHash]];
+
+          return [bucketHash, modsByBucket[bucketHash] ?? []];
```

Comment:
this needed to be set or a failed sync could cause incorrect types in the overall object